pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -85,6 +86,7 @@ buildPythonApplication rec {
     go_1_23
     fontconfig
   ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    makeBinaryWrapper
```

Comment:
IMO you can do this for Linux, too, as `makeBinaryWrapper` will generally reduce startup time